pkg/cli/root.go

```diff
@@ -408,13 +408,13 @@ func LoadConfiguration(config *config.Config, configPath string) error {
 	return nil
 }
 
-func isDefaultPolicyConfig(cfg *config.Config) bool {
+func isAnonymousPolicyConfig(cfg *config.Config) bool {
 	adminPolicy := cfg.AccessControl.AdminPolicy
 
-	log.Info().Msg("checking if default authorization is possible")
+	log.Info().Msg("checking if anonymous authorization is possible")
 
 	if len(adminPolicy.Actions)+len(adminPolicy.Users) > 0 {
-		log.Info().Msg("admin policy detected, default authorization disabled")
+		log.Info().Msg("admin policy detected, anonymous authorization disabled")
```

Comment:
`log.Info().Msg("admin policy detected, anonymous authorization is not the only authorization policy configured")`.

Similar updates for the other log messages.